java/src/main/java/ola/OlaClient.java

```diff
@@ -130,6 +142,38 @@ public PluginDescriptionReply getPluginDescription(int pluginId) {
     }
 
 
+    /**
+     * Return the state for a plugin.
```

Comment:
I think this should be:
```suggestion
     * Return the state of a plugin.
```